README.md

```diff
@@ -1,30 +1,26 @@
-# Outline Viewer Project
+[![Build Status](https://travis-ci.org/wpilibsuite/OutlineViewer.svg?branch=master)](https://travis-ci.org/wpilibsuite/OutlineViewer) [![codecov](https://codecov.io/gh/wpilibsuite/OutlineViewer/branch/master/graph/badge.svg)](https://codecov.io/gh/wpilibsuite/OutlineViewer)
```

Comment:
You can put these on different lines.
Easier to update/read in a diff.

They will show up on the same line in any markdown viewer.